test/generate_ssl_tests.pl

```diff
@@ -43,6 +43,7 @@ sub print_templates {
     # Add the implicit base configuration.
     foreach my $test (@ssltests::tests) {
         $test->{"server"} = { (%ssltests::base_server, %{$test->{"server"}}) };
+	$test->{"server2"} = { (%ssltests::base_server, %{$test->{"server2"}}) };
```

Comment:
Yeah, the whole "server2" was the tricky part, and because of the template, it ended up being easier to add the section to each test. I agree it would be nice to not have to define server2 each time. The "ServerName" indicator basically that SNI flag.